src/PerimeterxS2SValidator.php

```diff
@@ -131,6 +131,11 @@ public function verify()
             $this->pxCtx->setUuid($response->uuid);
             $this->pxCtx->setBlockAction($response->action);
             $this->pxCtx->setResponseBlockAction($response->action);
+            if(isset($response->data_enrichment)) {
+                $this->pxConfig['logger']->debug("{$response->data_enrichment->timestamp}");
```

Comment:
why are we logging the timestamp?

Reply:
was left there by mistake, thanks for the catch